volto/Dockerfile

```diff
@@ -14,7 +14,7 @@ COPY --chown=node . /build/
 ENV BUILD_TARGET=server
 RUN yarn global add mrs-developer
 RUN yarn develop
-RUN yarn install --frozen-lockfile
+RUN yarn install --frozen-lockfile && yarn cache clean
```

Comment:
As we're using multi-stage builds now, this will probably just slow things down again.